src/BinDeps.jl

```diff
@@ -48,7 +48,7 @@ module BinDeps
         if downloadcmd === nothing
             for download_engine in @windows? (:powershell, :curl, :wget, :fetch) : (:curl, :wget, :fetch)
                 if download_engine == :powershell
-                    checkcmd = `$download_engine -help`
+                    checkcmd = `$download_engine -NoProfile -Command "\$no_op"`
```

Comment:
The `"\$no_op"` is ugly and confusing. Just change this to `-Command ""` and I'll merge it.